contracts/CO30_Felix_Routine.tem_kujari/mission.sqm

```diff
@@ -0,0 +1,6237 @@
+			class Attributes
+			{
+				name="countdownIED1";
+				sizeA=200;
+				sizeB=200;
+				timeout[]={1800,1800,1800};
+				activationBy="ANYPLAYER";
+				type="SWITCH";
+				isServerOnly=1;
+			};
+			id=107;
+			type="EmptyDetectorAreaR250";
+			atlOffset=-9.5367432e-007;
+		};
+		class Item37
+		{
+			dataType="Group";
+			side="Independent";
+			class Entities
+			{
+				items=3;
+				class Item0
+				{
+					dataType="Object";
+					class PositionInfo
+					{
+						position[]={13006.063,13.083127,14091.575};
+						angles[]={0,3.8801808,0};
+					};
+					side="Independent";
+					flags=7;
+					class Attributes
+					{
+						name="IED2";
+						class Inventory
+						{
+							class uniform
+							{
+								typeName="U_C_Poloshirt_blue";
+								isBackpack=0;
+							};
+							class vest
+							{
+								typeName="V_TacChestrig_cbr_F";
+								isBackpack=0;
+								class MagazineCargo
+								{
+									items=1;
+									class Item0
+									{
+										name="DemoCharge_Remote_Mag";
+										count=2;
+										ammoLeft=1;
+									};
+								};
+							};
+							map="ItemMap";
+							compass="ItemCompass";
+							watch="ItemWatch";
+							headgear="H_ShemagOpen_tan";
+						};
+					};
+					id=109;
+					type="C_man_polo_6_F_afro";
+					atlOffset=0.00010108948;
+					class CustomAttributes
+					{
+						class Attribute0
+						{
+							property="pitch";
+							expression="_this setpitch _value;";
+							class Value
+							{
+								class data
+								{
+									singleType="SCALAR";
+									value=1.01;
+								};
+							};
+						};
+						nAttributes=1;
+					};
+				};
+				class Item1
+				{
+					dataType="Waypoint";
+					loiterAltitude=-3.4028235e+038;
+					position[]={13004.722,12.839472,14091.722};
+					type="Hold";
+					class Effects
+					{
+					};
+					showWP="NEVER";
+					id=110;
+					atlOffset=1.4305115e-005;
+				};
+				class Item2
+				{
+					dataType="Waypoint";
+					loiterAltitude=-3.4028235e+038;
+					position[]={12874.638,12.466481,14117.979};
+					combatMode="RED";
+					speed="FULL";
+					combat="CARELESS";
+					class Effects
+					{
+					};
+					showWP="NEVER";
+					id=111;
+					type="Move";
+					atlOffset=-9.5367432e-007;
+				};
+			};
+			class Attributes
+			{
+			};
+			id=108;
+			atlOffset=0.00010108948;
+		};
+		class Item38
+		{
+			dataType="Trigger";
+			position[]={12910.051,12.978206,14106.216};
+			class Attributes
+			{
+				name="countdownIED2";
+				sizeA=200;
+				sizeB=200;
+				timeout[]={3000,3000,3000};
+				activationBy="ANYPLAYER";
+				type="SWITCH";
+				isServerOnly=1;
+			};
+			id=112;
+			type="EmptyDetectorAreaR250";
+			atlOffset=9.5367432e-007;
+		};
+		class Item39
+		{
+			dataType="Trigger";
+			position[]={12908.656,13.198047,14157.485};
+			class Attributes
+			{
+				name="detoneIED1";
+				condition="this && IED1 in thisList && !(IED1 getVariable [""ace_captives_isHandcuffed"", false])";
```

Comment:
Should you maybe put a detonator or dead man switch in their inventory and also check for that. Give the players a strong indication of the presence of an IED, and also then if they release the unit after removing the detonator they won't still blow up